scripts/development/m3_stack/start_m3.sh

```diff
@@ -26,10 +27,28 @@ if [[ "$USE_JAEGER" = true ]] ; then
     fi
 fi
 
-docker-compose -f docker-compose.yml up $DOCKER_ARGS m3coordinator01
-docker-compose -f docker-compose.yml up $DOCKER_ARGS m3db_seed
-docker-compose -f docker-compose.yml up $DOCKER_ARGS prometheus01
-docker-compose -f docker-compose.yml up $DOCKER_ARGS grafana
+# Use standard coordinator config
+cp ./m3coordinator-standard.yml ${RELATIVE}/bin/m3coordinator.yml
+
+if [[ "$FORCE_BUILD" == true ]] || [[ "$BUILD_M3COORDINATOR" == true ]]; then
+    prepare_build_cmd "make m3coordinator-linux-amd64"
```

Comment:
Hmmm, so you'll always have to run it with `FORCE_BUILD=true` the first time, right?

Reply:
Yeah true, maybe I can try to see if the image is built or not yet.